config/locales/ru.yml

```diff
@@ -1096,7 +1097,8 @@ ru:
     reference: "Ссылка"
     refund: "Возврат"
     refund_amount_must_be_greater_than_zero:
-    refund_reasons: "Причины возврата"
+    refund_reasons: "Причины возврата денег"
+    return_reasons: "Причины возврата товара"
```

Comment:
@connecticus would you mind to check the order again? As commented above, your proposed changes are not reflected here on GitHub.